### PR TITLE
console: one last command->command2

### PR DESCRIPTION
**Pull request by bradjc**

### CODE REVIEW

**Review by hudson-ayers — APPROVED** [review]

**Review by ppannuto — APPROVED** [review]
bors r+

File: libtock/console.c [hunk 1]
```diff
@@ -163,5 +163,13 @@ int getch(void) {
...
 
 int getnstr_abort(void) {
-  return command(DRIVER_NUM_CONSOLE, 3, 0, 0);
+  syscall_return_t com = command2(DRIVER_NUM_CONSOLE, 3, 0, 0);
+  if (com.type == TOCK_SYSCALL_SUCCESS) {
+    return TOCK_SUCCESS;
+  } else if (com.type > TOCK_SYSCALL_SUCCESS) {
+    // Returned an incorrect success code
+    return TOCK_FAIL;
+  } else {
```
Comment (phil-levis) [inline libtock/console.c:172]: Please note that this is not quite correct: if console returns a different failure code than expected, this doesn't return TOCK_EBADVAL. I'll patch in #177 .

Comment (hudson-ayers) [reply libtock/console.c:172]: Assuming the failure type seems to have happened in the majority of the transitioned capsules. From a very quick look:
- UDP
- alarm_internal
- spi_peripheral
- adc
- ble
- sdcard
- gpio_async
- alarm_timer
- ninedof
- 15.4
- crc
- analog comparator
- ... I stopped looking at this point but a bunch more

Comment (hudson-ayers) [reply libtock/console.c:172]: in fact, the only place that `EBADRVAL` is constructed is in `app_state` and `nonvolatile_storage`

**Pull request merged**